### PR TITLE
Migrate Visualization->"All parts of curtain wall visible" to 3.0

### PR DESCRIPTION
**Pull request by leosobral**

### CODE REVIEW

**Review by cschladetsch — COMMENTED** [review]
Punctuation at end of paragraphs and bullet points is inconsistent.

There is unneeded vertical white space.

File: Test Cases/v3.0/Visualization/All parts of curtain wall visible/unzipped/bcf.version [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (jasollien) [inline Test Cases/v3.0/Visualization/All parts of curtain wall visible/unzipped/bcf.version:1]: Can you do a new export. Detailed version should not be here

**Comment by jasollien** [comment]
Missing extensions.xml

File: Test Cases/v3.0/Visualization/All parts of curtain wall visible/unzipped/ee9a9498-698b-44ed-8ece-b3ae3b480a90/markup.bcf [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
+    </Header>
+    <Topic Guid="ee9a9498-698b-44ed-8ece-b3ae3b480a90" ServerAssignedId="826" TopicStatus="Open">
```
Comment (ykulbak) [inline Test Cases/v3.0/Visualization/All parts of curtain wall visible/unzipped/ee9a9498-698b-44ed-8ece-b3ae3b480a90/markup.bcf:6]: Please add TopicType

**Review by ykulbak — COMMENTED** [review]
The bcf imports correctly on Aconex

File: Test Cases/v3.0/Visualization/All parts of curtain wall visible/unzipped/bcf.version [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+<Version xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" VersionId="3.0" xsi:schemaLocation="version.xsd">
+    <DetailedVersion>3.0</DetailedVersion>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/All parts of curtain wall visible/unzipped/bcf.version:3]: Remove detailed version

**Comment by pasi-paasiala** [comment]
Imports fine in Solibri.

**Review by pasi-paasiala — APPROVED** [review]
+1

**Pull request merged**